atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ProfilingCassandraClient.java

```diff
@@ -19,6 +19,7 @@
 import com.palantir.atlasdb.keyvalue.cassandra.thrift.ThriftObjectSizeUtils;
 import com.palantir.atlasdb.logging.KvsProfilingLogger;
 import com.palantir.atlasdb.logging.LoggingArgs;
+import com.palantir.common.base.CallableCheckedException;
```

Comment:
I did this refactor here because a) it's well contained, obvious and very few lines of code, and b) moving this interface to the shared library in a separate PR would lose some context of why that's happening.